plugins/woocommerce/client/legacy/js/frontend/a8c-address-autocomplete-service.js

```
@@ -0,0 +1,460 @@
+						console.error(
+							'Automattic Address Suggestion: Missing address ID'
+						);
+						return null;
+					case 'no_place':
+						console.error(
+							'Automattic Address Suggestion: No place found'
+						);
+						return null;
+					case 'missing_session_id':
+						sessionId =
+							crypto && crypto.randomUUID
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)